package.json

```diff
@@ -5,7 +5,12 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon --exec babel-node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --forceExit --verbose",
+    "coverage": "jest --forceExit --coverage && cat ./coverage/lcov.info | coveralls",
```

Comment:
I think we have to change this script before we merge
It's running the tests twice
And its complaining that we don't have coveralls installed.

Reply:
I'm on it

Reply:
Wait, that's your job na Jude